src/services/api-claim-service.ts

```diff
@@ -24,11 +27,26 @@ export class ApiClaimService {
     }
 
     this.addApiClaim(mfeName, apiClaim)
-
     this.updateMfeConfigApi(mfeName, apiClaim.name, serviceUrl)
   }
 
-  private addApiClaim(mfeName: string, apiClaim: ApiClaim): void {
+  public addExternalApiClaim(
+    mfeName: string,
+    apiClaim: ExternalApiClaim
+  ): void {
+    const url = this.cmService.getBundleMicroserviceUrl(
```

Comment:
what if url is null or undefined (because bundleId and serviceId are invalid)? I guess connection issues will be handled by cmService instead by url will be null for sure

Reply:
I presumed that the api would respond with an error if bundleId and serviceId are invalid or non-existent, which would then cause the cli to throw an error as well (will be implemented in ENG-3708). If indeed it just returns a null value, I'll make the necessary changes to handle it.